src/common/transformations/src/transformations/op_conversions/convert_interpolate11_downgrade.cpp

```diff
@@ -11,6 +11,42 @@
 #include <openvino/opsets/opset4.hpp>
 
 #include "itt.hpp"
+#include "utils.hpp"
+
+namespace {
+// v4_sizes, v4_scales
+std::pair<ov::Output<ov::Node>, ov::Output<ov::Node>> make_v4_inputs(
+    const std::shared_ptr<ov::opset11::Interpolate>& interpolate) {
+    ov::pass::NodeRegistry registry;
+    std::pair<ov::Output<ov::Node>, ov::Output<ov::Node>> ret;
+    std::shared_ptr<ov::Node> broadcast_shape;
+
+    if (interpolate->get_input_size() == 3) {
+        // broadcast dummy constant to the shape of axes
+        broadcast_shape = registry.make<ov::opset4::ShapeOf>(interpolate->input_value(2));
+    } else {
+        // broadcast dummy constant to the rank of data
+        broadcast_shape = registry.make<ov::opset4::ShapeOf>(interpolate->input_value(0));
+        broadcast_shape = registry.make<ov::opset4::ShapeOf>(broadcast_shape);
+    }
+
+    if (interpolate->get_attrs().shape_calculation_mode == ov::op::util::InterpolateBase::ShapeCalcMode::SCALES) {
+        ret.second = interpolate->input_value(1);
+        std::shared_ptr<ov::Node> sizes_input = ov::opset4::Constant::create(ov::element::i32, ov::Shape{}, {1});
+        sizes_input = registry.make<ov::opset4::Broadcast>(sizes_input, broadcast_shape);
+        ret.first = sizes_input;
+    } else {
+        ret.first = interpolate->input_value(1);
+        std::shared_ptr<ov::Node> scales_input = ov::opset4::Constant::create(ov::element::f32, ov::Shape{}, {1.0f});
+        scales_input = registry.make<ov::opset4::Broadcast>(scales_input, broadcast_shape);
```

Comment:
As in the referenced example:
https://github.com/openvinotoolkit/openvino/blob/1eb6ad20c382c6412d424a172e7a9d60b377f5b8/src/common/transformations/src/transformations/op_conversions/unique_decomposition.cpp#L63-L64
the Constant ops should be registered as well:
```suggestion
    if (interpolate->get_attrs().shape_calculation_mode == ov::op::util::InterpolateBase::ShapeCalcMode::SCALES) {
        ret.second = interpolate->input_value(1);
        std::shared_ptr<ov::Node> sizes_input = rg.make<ov::opset4::Constant>(element::i32, Shape{}, {1});
        sizes_input = registry.make<ov::opset4::Broadcast>(sizes_input, broadcast_shape);
        ret.first = sizes_input;
    } else {
        ret.first = interpolate->input_value(1);
        std::shared_ptr<ov::Node> scales_input =  rg.make<ov::opset4::Constant>(element::f32, Shape{}, {1.0f});
        scales_input = registry.make<ov::opset4::Broadcast>(scales_input, broadcast_shape);
```